app/src/main/java/woowacourse/payments/ui/cardlist/composable/RegisterPaymentCard.kt

```diff
@@ -23,11 +23,23 @@ fun RegisterPaymentCard(
     card: Card,
     modifier: Modifier = Modifier,
 ) {
-    Column(modifier = modifier.padding(horizontal = 16.dp)) {
+    Column(
+        modifier =
+            modifier
+                .fillMaxSize()
+                .background(Color(card.bank.color))
+                .padding(horizontal = 16.dp),
+    ) {
+        Text(
+            modifier = Modifier.padding(top = 14.dp),
+            text = card.bank.bankName,
+            color = Color(0xFFFFFFFF),
```

Comment:
```suggestion
            color = Color.White,
```
컴포즈에서 제공해주는 색상을 사용해볼 수도 있을 것 같아요 !

Reply:
[972420f](https://github.com/woowacourse/android-payments/pull/80/commits/972420f784c117be8e2063242d53370b75f53587)

알았는데도 쓰지않은.. 반성해야겠네요...